### DISCUSSION TITLE
About upcoming online meetups

### DISCUSSION THREAD

**Discussion by sbibek086** [sbibek086/Git-and-Github-Guides#11] [Announcements]
Upcoming online meetups will be in October 15, 2022, 8 PM (GMT +5:45) about Excelsheet jobs and how to make most of it? Please bookmark.
in https://meet.google.com/stb-zjnv-ghe

**Comment by KiranBhatta** [comment]
please provide the link

**Reply by sbibek086** [reply]
its in meet.google.com/stb-zjnv-ghe. please read above
